### ISSUE TITLE
[Question] notify is pretty aggressive with every action.

### ISSUE THREAD

**Issue by bassamsdata** [rcarriga/nvim-notify#169]
Firstly, I really liked the plugin and thanks for your efforts.

I'm new to `lua` so I have some issues that I'm not been able to tackle and I appreciate any help.

1. I need to reduce the amount that the notification stays on screen (I changed the `timeout` to 500 and still the notification stays a long time on screen. I've read the docs but didn't able to solve the issue)
2. too many notifications. (I want to disable some basic notifications to be pushed to notify)

Code I'm using is below for timeout:
For the number of notifications: I noticed that notify is sending me a lot of notifications about things shouldn't be always pushed to notify like:
1. yanked lines. (I set `vim.opt.report` to high number to get rid of these notification but other things I couldn't do much)
4. pasted lines
5. changes made
6. fewer lines
7. etc..
I think below gif describe what I'm taking about.

https://user-images.githubusercontent.com/105807570/214161048-fffade9e-6aed-4bc3-811e-5da6cc4dad7f.mov

I'm not sure if this is the default behavior or not.

Thank again

**Comment by rcarriga** [comment]
The timeout is how long the notification stays once it has reached the timed stage so just set the timeout to something like `50` to be very quick. Or use the `static` stages. More details here https://github.com/rcarriga/nvim-notify#changing-the-window

nvim-notify doesn't send any notifications itself, only plugins that use it. What you're seeing is noice.nvim taking over the output of commands like `:echo` so you have to change your noice config to prevent that by disabling messages
```lua
    messages = {
      enabled = false, -- enables the Noice messages UI
    }
```

**Comment by bassamsdata** [comment]
Oh thank you for your answer. I didn't know that `noice` is doing all of that. the static stages is really helpful and indeed it feels that timeout is less. 

Yet disabling the messages in `noice` didn't work correctly, because If I set it, it disable all other features that I have set. (I don't why) but that's a different story related to `noice` which I probably delete since it has a loooooot of details in the documentations hahaha and I don't wanna go into a rabbit hole. 

Thanks again for the help
